src/pages/leejiyon/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  onChangeInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  render() {
+    const validation = this.state.id.includes('@') && this.state.pw.length > 4;
+    return (
+      <div className="LoginJY">
+        <main className="LoginBox">
+          <h1 className="title">Westagram</h1>
+          <div className="login">
+            <input
+              className="loginInputs loginId"
+              name="id"
+              value={this.state.id}
+              onChange={this.onChangeInput}
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+            />
+            <input
+              className="loginInputs loginPw"
+              name="pw"
+              value={this.state.pw}
+              onChange={this.onChangeInput}
+              type="password"
+              placeholder="비밀번호"
+            />
+          </div>
+
+          <Link to="/mainjy">
+            {validation ? (
+              <button className="loginBtn blueBackgroundColor">로그인</button>
+            ) : (
+              <button className="loginBtn ">로그인</button>
+            )}
+          </Link>
+          <a
+            className="findPw"
+            href="https://www.instagram.com/accounts/password/reset/"
+          >
+            비밀번호를 잊으셨나요?
+          </a>
```

Comment:
리액트는 효율적인 렌더링을 위해 a태그보다 Link태그의 사용을 권장하고 있다고 합니다 ! 
```suggestion
          <Link to="/https://www.instagram.com/accounts/password/reset/"
          >
            비밀번호를 잊으셨나요?
          </Link>
```

Reply:
앗 이거는 실제 인스타 사이트로 연동되게끔 했습니다 !

Reply:
외부링크로 이동하실땐 a tag 사용하신것 좋습니다!